src/idl_gen_kotlin.cpp

```diff
@@ -1218,18 +1219,21 @@ class KotlinGenerator : public BaseGenerator {
   }
 
   static void GenerateGetRootAsAccessors(const std::string &struct_name,
-                                         CodeWriter &writer) {
+                                         CodeWriter &writer,
+                                         bool useJvmStaticAnnotation) {
```

Comment:
why don't you pass an `IDLOptions &` here instead of a `bool`, which simplifies all the callers a bit and is more general? Same for the other functions.

Reply:
@aardappel Only few parts replaced to IDLOptions because some of case one method should be proceed differently.
Kotlin compiler allow suffix @JvmStaticAnnotation to methods in companion object.